pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,86 +1,99 @@
+#!bash
```

Comment:
That shebang doesn't conform to POSIX. The interpreter must be an absolute path.